tools/eslint-rules/non-ascii-character.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileOverview Any non-ASCII characters in lib/ will increase the size
+ *               of the compiled node binary. This linter rule ensures that
+ *               any such character is reported.
+ * @author Sarat Addepalli <sarat.addepalli@gmail.com>
+ */
+
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const nonAsciiRegexPattern = new RegExp(/[^\r\n\x20-\x7e]/);
```

Comment:
Any reason not to use a RegExp literal like this?: 

```javascript
const nonAsciiRegexPattern = /[^\r\n\x20-\x7e]/;
```

Seems more readable to me and also more in line with our general coding style. (I'm kind of surprised this isn't caught by a lint rule itself, to be honest. Or maybe it is?)